src/pages/gen2/build-a-backend/add-aws-services/custom-resources/index.mdx

```diff
@@ -200,169 +230,32 @@ const sendEmail = async (msg: Message) => {
 const deleteMsg = async (receiptHandle: string, topicArn: string) => {
   const command = new DeleteMessageCommand({
     ReceiptHandle: receiptHandle,
-    TopicArn: topicArn,
+    TopicArn: topicArn
   });
 
   try {
     await snsClient.send(command);
   } catch (err) {
-    console.error("Error deleting message", err);
+    console.error('Error deleting message', err);
   }
 };
 ```
 
-The `CustomNotifications` CDK construct can then be added to the `backend` of the Amplify stack one or more times, with different props for each instance.
-
-```ts title="amplify/backend.ts"
-import { defineBackend } from '@aws-amplify/backend';
-import { CustomNotifications } from './custom/CustomNotifications/resource';
-import { data } from './data/resource';
-
-const backend = new defineBackend({
-  data,
-});
-
-new CustomNotifications(
-  backend.getStack("CustomNotifications"),
-  "CustomNotifications",
-  { sourceAddress: "sender@example.com", }
-);
-```
-
-<Callout>
-**Note:** Resources created by the `define*` helpers can only be accessed _after_ setting them on the `backend`.
-</Callout>
-
-## Defining a CDK Stack
```

Comment:
This example was not entirely technically correct and also too long. I think just showing how to use existing CDK Constructs and make a custom construct is sufficient for this docs page.